data/public_suffix_list.dat

```diff
@@ -2,6 +2,11 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+// Please pull this list from, and only from https://publicsuffix.org/list/public_suffix_list.dat , 
+// rather than any other VCS sites. Pulling from any other URL is not guaranteed to be supported.
```

Comment:
Given this new (and apparently very timely) comment, maybe it's worth pinging the public suffix team about the default download location causing issues with OTP 19.1. They clearly don't want us downloading the list from somewhere else. I don't know if the issues we're seeing are a problem with OTP or with their servers, but perhaps they'd be willing to help look into it?